auth-mailchimp-sync/functions/src/index.ts

```diff
@@ -95,5 +99,83 @@ export const removeUserFromList = functions.handler.auth.user.onDelete(
     } catch (err) {
       logs.errorRemoveUser(err);
     }
-  }
-);
+  });
+
+export const addExistingUsersToList = functions.tasks
+  .taskQueue()
+  .onDispatch(async (data) => {
+    const runtime = getExtensions().runtime();
+    if (!config.doBackfill) {
+      await runtime.setProcessingState(
+        "PROCESSING_COMPLETE",
+        "No existing documents imported into BigQuery."
```

Comment:
Optional: I think "No processing requested" would be better. So it's not confused for the case where there are no existing users.